ui-react/packages/atlasmap-core/src/services/mapping-management.service.ts

```diff
@@ -579,6 +579,15 @@ export class MappingManagementService {
     }
   }
 
+  conditionalMappingExpressionEnabled(): boolean {
+    if (!this.cfg || !this.cfg.mappings || !this.cfg.mappings.activeMapping) {
+      return false;
+    };
+    return (this.cfg.mappings && this.cfg.mappings.activeMapping &&
```

Comment:
This return can be simplified to `return !!(this.cfg.mappings?.activeMapping?.transition?.enableExpression?)` with the new TS